src/binding.ts

```diff
@@ -47,6 +48,38 @@ export class BindingService {
     return BindingService._instance;
   }
 
+  async editBinding(workspaceFolderItem: WorkspaceFolderItem): Promise<void> {
+    const workspaceFolder = VSCode.workspace.workspaceFolders.find(f => f.name === workspaceFolderItem.name);
+    const selectedFolderName = workspaceFolderItem.name;
+    const connectionId = workspaceFolderItem.connectionId;
+    const remoteProjects =
+      await this.getRemoteProjectsItems(connectionId, workspaceFolder, workspaceFolderItem.serverType);
+
+    if (remoteProjects) {
+      const remoteProjectsQuickPick = VSCode.window.createQuickPick();
+      remoteProjectsQuickPick.title =
+        `Select Project to Bind with '${selectedFolderName}/'`;
+      remoteProjectsQuickPick.placeholder =
+        `Select the remote project you want to bind with '${selectedFolderName}/' folder`;
+      remoteProjectsQuickPick.items = remoteProjects;
+      remoteProjectsQuickPick.ignoreFocusOut = true;
+
+      remoteProjectsQuickPick.onDidChangeSelection(selection => {
+        const selectedRemoteProject = selection[0];
+        this.saveBinding(selectedRemoteProject.description, connectionId, workspaceFolder);
+        VSCode.window.showInformationMessage(`Workspace folder '${selectedFolderName}/'
+                has been bound with project '${selectedRemoteProject.label}'`);
+        remoteProjectsQuickPick.dispose();
+      });
+      remoteProjectsQuickPick.show();
```

Comment:
handling of `onDidTriggerItemButton` event is missing. This means remote projects cannot be opened in SQ/SC when clicking the relevant button.